src/components/NFTViewCard.vue

```diff
@@ -122,6 +122,18 @@ export default defineComponent({
     ModalWindow,
     VueJsonPretty,
   },
+
+  methods: {
+    readTicketAttributeValue (ticket: INFT, trait_type_item: string) {
+       // Takes in a ticket and returns a particular attribute trait type value
+       let attr = ticket.metadataExternal.attributes.find((tt: { trait_type: string, value: string; }) => tt.trait_type == trait_type_item)
+       if (attr){
```

Comment:
quick note here:

1) I rewrote this to use .find() , but still need the if evaluation writing this with a ternary expression inside the {{}} expression made the syntax harder to understand

---i'm checking if there's a way to automatically convert an 'undefined' value returned by .find() to a specific value instead of having to evaluate if() for that to happen

Reply:
hmm this seems to be a nicer way to do it:

       return typeof attr != 'undefined' ? attr.value : "Attribute Not Set"

and avoid the if()